pkg/kubelet/dockertools/manager.go

```diff
@@ -568,6 +551,11 @@ func (dm *DockerManager) GetPodStatus(pod *api.Pod) (*api.PodStatus, error) {
 			// values if possible.
 			containerStatus.RestartCount = oldStatus.RestartCount
 			containerStatus.LastTerminationState = oldStatus.LastTerminationState
+		} else {
+			// A container in manifest but not in docker list and oldStatuses,
+			// it seems that it is sure that this is a new container.
+			// (taotaotheripper) Checkout whether this means a new container
+			containerStatus.RestartCount = -1
```

Comment:
A -1 RestartCount seems counter-intuitive and may cause user confusion. 

I think the restart count should be zero if the container has only been started once. In other words, the restart count read from the docker label should apply only if current state is terminated or if LastTerminationState is non-nil.